src/Game.cpp

```diff
@@ -289,14 +289,6 @@ void Game::TimeStep(float step)
 		if (Pi::game->GetTime() >= m_hyperspaceEndTime) {
 			SwitchToNormalSpace();
 			m_player->EnterSystem();
-			// event "onEnterSystem(player)" is in the event queue after p_player->EnterSystem()
-			// in this callback, for example, fuel is reduced
-			// but event handling has already passed, and will only be in the next frame
-			// it turns out that we are already in the system, but the fuel has not yet been reduced
-			// and then the drawing of the views will go, and inconsistencies may occur,
-			// for example, the sphere of the hyperjump range will be too large
-			// so we forcefully call event handling again
-			LuaEvent::Emit();
```

Comment:
What was changed to make this comment and call no longer necessary? I don't need it documented in the code, but I'd appreciate an explanation of what resolved this rather long comment about an edge case.

Reply:
@Web-eWorks The purpose for which the `LuaEvent::Emit()` call was added is described in the comment. Now it turns out that _all tradeships are generated in this call_. Вut in this frame, delta_t is in effect, which was in effect during the hyperjump. In the same frame, the next position of these ships is calculated, based on this _huge_ delta_t. This leads to the fact that in the very first frame the positions of the ships are very displaced from the positions where they were generated, and the speeds turn out to be too high, therefore, some of the ships that are generated near the destination point may even fly by. We didn't notice it before, because tradeships spawned at zero speeds.

I thought that the first problem is not as bad as the second, and in order not to dig into the solution of the first problem, I just accept it. Just if you get out of the hyperjump and you have the sector map turned on, your jump range sphere will be displayed too large in the first frame, it will decrease to the correct size only in the next frame, that's the whole problem.